<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStadium</class>
 <widget class="QDialog" name="ModifyStadium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="stadium_name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stadium Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="team_name_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="street_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Street</string>
   </property>
  </widget>
  <widget class="QLabel" name="city_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_number_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="capacity_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacity</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="stadium_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="team_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="street">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="city">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone_number">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="date">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="capacity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
